apps/staking/src/hooks/use-api.tsx

```diff
@@ -195,13 +195,15 @@ const useApiContext = (
               publicKey: wallet.publicKey,
               signAllTransactions: wallet.signAllTransactions,
               signTransaction: wallet.signTransaction,
```

Comment:
`signTransaction` and `signAllTransactions` are not used anymore?

Reply:
Yes they are not used afaik, but I thought it's better to keep them.